templates/server-ha-active-service.yaml

```diff
@@ -15,6 +15,11 @@ metadata:
   annotations:
 {{ template "vault.service.annotations" .}}
 spec:
+  {{- if and (.Values.server.service.externalTrafficPolicy ) (print .Values.server.service.externalTrafficPolicy | lower | title | eq "Local"  )}}
```

Comment:
I believe this setting is only used when the service type is `LoadBalancer`, so an additional check might be needed.

Reply:
This configuration is valid for NodePort and it allows you to get the sourceIP of the end user -
https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport
Thia also works when running with GKE as per the document I've referenced.
With this information in hand do you believe a change should be made?

Reply:
I think this still needs additional logic to check for the service type. For example the default is ClusterIP so this setting wouldn't be used by that. Thoughts?

Reply:
I think that the simplest to implement addition will be to take care only the NodePort + externalTrafficPolicy=Local case
Just pushed the change